other/replace-image-registry/pods.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod01
+  namespace: replace-registry-ns
+spec:
+  containers:
+  - name: busybox
+    image: busybox:1.35
+  - name: busybox02
+    image: docker.io/busybox:1.35
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod02
+  namespace: replace-registry-ns
+spec:
+  initContainers:
+  - name: kyverno
+    image: kyverno:1.10.1
+  - name: busybox-init
+    image: docker.io/busybox:1.35
+  containers:
+  - name: busybox
+    image: registry.corp.com/kyverno:1.10.1
+  - name: busybox02
+    image: docker.io/busybox:1.35
```

Comment:
Can you also add a longer path here, something like `ghcr.io/foo/bar/baz/busybox:1.35`?

Reply:
I'm guessing in this case the regex should be `myregistry.corp.com/foo/bar/baz/busybox:1.35`. That is the output of the policy as well.

Reply:
Yes